src/styles/themes/_default-theme.scss

```diff
@@ -6,201 +6,291 @@
   --disruptive-color-50: #e46f6f;
-  --disruptive-color-60: #c45353; // $color-tertiary-disruptive (Graphics)
-  --disruptive-color-70: #993838; // $color-secondary-disruptive (Text)
-  --disruptive-color-80: #6c2222; // $color-primary-disruptive (Extra high contrast)
+  --disruptive-color-60: #c15151;
+  --disruptive-color-70: #993838;
+  --disruptive-color-80: #6c2222;
   --disruptive-color-90: #3b1010;
   --disruptive-color-100: #0a0202;
-  --disruptive-color: #6c2222; // $color-primary-disruptive (Extra high contrast)
+
+  --disruptive-background1-color: var(--disruptive-color-10); // Surface 1
+  --disruptive-background2-color: var(--disruptive-color-20); // Surface 2
+  --disruptive-background3-color: var(--disruptive-color-30); // Surface 3
+  --disruptive-tertiary-color: var(--disruptive-color-60); // Graphics
+  --disruptive-secondary-color: var(--disruptive-color-70); // Text
+  --disruptive-color: var(--disruptive-color-80); // Extra high contrast
 
   // Red-Orange base colors
-  --redorange-color-10: #fff2ed; // $color-background1-red-orange (Surface 1)
-  --redorange-color-20: #ffd1bd; // $color-background2-red-orange (Surface 2)
-  --redorange-color-30: #ffb396; // $color-background3-red-orange (Surface 3)
+  --redorange-color-10: #fff2ed;
+  --redorange-color-20: #ffd1bd;
+  --redorange-color-30: #ffb396;
   --redorange-color-40: #ff9977;
   --redorange-color-50: #f8845f;
-  --redorange-color-60: #d26743; // $color-tertiary-red-orange (Graphics)
-  --redorange-color-70: #a54d2c; // $color-secondary-red-orange (Text)
-  --redorange-color-80: #78351a; // $color-primary-red-orange (Extra high contrast)
+  --redorange-color-60: #d26743;
+  --redorange-color-70: #a54d2c;
+  --redorange-color-80: #78351a;
   --redorange-color-90: #4c1f0c;
   --redorange-color-100: #1f0c03;
-  --redorange-color: #78351a; // $color-primary-red-orange (Extra high contrast)
+
+  --redorange-background1-color: var(--redorange-color-10); // Surface 1
+  --redorange-background2-color: var(--redorange-color-20); // Surface 2
+  --redorange-background3-color: var(--redorange-color-30); // Surface 3
+  --redorange-tertiary-color: var(--redorange-color-60); // Graphics
+  --redorange-secondary-color: var(--redorange-color-70); // Text
+  --redorange-color: var(--redorange-color-80); // Extra high contrast
 
   // Orange base colors
-  --orange-color-10: #fff8eb; // $color-background1-orange (Surface 1)
-  --orange-color-20: #ffe3b0; // $color-background2-orange (Surface 2)
-  --orange-color-30: #ffcd78; // $color-background3-orange (Surface 3)
+  --orange-color-10: #fff8eb;
+  --orange-color-20: #ffe3b0;
+  --orange-color-30: #ffcd78;
   --orange-color-40: #ffb650;
   --orange-color-50: #f29d31;
-  --orange-color-60: #ce811a; // $color-tertiary-orange (Graphics)
-  --orange-color-70: #9d6309; // $color-secondary-orange (Text)
-  --orange-color-80: #6c4500; // $color-primary-orange (Extra high contrast)
+  --orange-color-60: #c97e19;
```

Comment:
whats with the color tweaks? assuming this is per the spec

Reply:
Yes